package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kriya",
-  "version": "1.6.2",
+  "version": "1.6.3",
```

Comment:
This is a new feature, so it should be `1.7.0` if it doesn't break existing API usage.